.github/issue_template.md

```diff
@@ -0,0 +1,41 @@
+#### Things to consider
+
+1. Please check that you are using the latest scapy version, e.g. installed via:
+
+   `pip install https://github.com/secdev/scapy`
+
+2. If you are here to ask a question - please use gitter instead: https://gitter.im/secdev/scapy
+
+3. Please understand that **this is not a forum** but an issue tracker. Invalid issues will be closed without any comment.
```

Comment:
Nope.

Reply:
Sorry for wasting your time - but in the context of https://github.com/secdev/scapy/pull/1155#discussion_r173662485 - should I 
a. remove (3) or 
b. wait until you talked to the other ... deciders?

(I still don't know how this project is run, e.g. collaborative, democratic, dictatorial, super-hero-centric, ... so it is hard to apply a yes/no request here)